### PR TITLE
:book: display open closes times for week

### PR DESCRIPTION
**Pull request by markysoft**

### CODE REVIEW

File: test/unit/lib/formatOpeningTimes.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
+const chai = require('chai');
+const getOpeningTimesOverview = require('../../../app/lib/getOpeningTimesOverview');
+
+const expect = chai.expect;
+
+describe('getOpeningTimesOverview', () => {
+  it('should return first open and last closes for each day as hour and am/pm', () => {
+    const openingTimes = {
+      general: {
+        monday: [{ opens: '08:00', closes: '15:00' }, { opens: '16:00', closes: '17:00' }],
```
[the remark this answers is not in the document]
Comment (markysoft) [reply test/unit/lib/formatOpeningTimes.js]: I'll add another test for the scenario

**Comment by markysoft** [comment]
suggestions addressed, awaiting input from team prior to tackling the 24 hour opening message

File: app/lib/getOpeningTimesOverview.js [hunk 2]
```diff
@@ -19,7 +19,8 @@ function getDayOpeningTimes(sessions) {
     return {};
   }
   const firstOpen = sessions[0].opens;
-  const lastOpen = sessions[sessions.length - 1] && sessions[sessions.length - 1].closes;
+  const lastSession = sessions.length - 1;
```
Comment (st3v3nhunt) [inline app/lib/getOpeningTimesOverview.js]: Any reason why it couldn't be this:
```js
const lastSession = sessions[sessions.length -1];
const lastOpen = lastSession && lastSession.closes;
```

Comment (markysoft) [reply app/lib/getOpeningTimesOverview.js]: oops, will change it

**Pull request merged**